DirectEmployers/MyJobs
# [PD-3231] Make sitemap the jobs show up on more consistent, remove variable microseconds from the sitemap index lastmod time.

## What's this PR do?
Two things:
1. Adds microseconds to the lastmod in the sitemap index.
2. Switches pagination to use guids, instead of only sorting by date_new, so there's no chance that a job can move between pages. 

## How should this be manually tested?
yoursite.jobs/sitemap.xml should load, and have exactly 500 entries that look like this:
```
<sitemap>
    <loc>
        http://www.my.jobs/sitemap-2017-04-28.xml?page=012345
    </loc>
    <lastmod>2017-04-28T19:00:00</lastmod>
</sitemap>
<sitemap>
    <loc>
        http://www.my.jobs/sitemap-2017-04-28.xml?page=6789A
    </loc>
    <lastmod>2017-04-28T19:00:00</lastmod>
</sitemap>
```
confirm that ?page= is always a list of valid guid prefixes, and that the lastmod never has a microsecond value other than 00.

Navigating to one of the sitemap pages, all jobs on the page should have a date_new of that date and be prefixed with one of the characters specified in the page parameter. For example, if you went to http://www.my.jobs/sitemap-2017-04-28.xml?page=6789A you should get only jobs that were new on 4/28 with guids that start with 6, 7, 8, 9, or A.
